spec/models/comment_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  fixtures :users, :posts
+
+  describe 'associations' do
+    it { should belong_to(:user) }
+    it { should belong_to(:post) }
+  end
+
+  it 'update the comments_counter of the post' do
+    comment = Comment.create(user: users(:one), post: posts(:one), text: 'Hello')
+    expect(comment.post.comments_counter).to eq(6)
```

Comment:
* I get an error here when i try to run your tests , kindly fix it 
![image](https://github.com/Rebelzob/blog-app/assets/52497246/6c993374-7590-444a-978f-24d9a1f662e0)

Reply:
@Davidosky007 Thanks for the review!

actually in my terminal, if I change the text I get a different output.
![image](https://github.com/Rebelzob/blog-app/assets/52711805/2a1a0c38-846b-444a-94d4-9097c5c25e09)